dashboard/src/components/ui/v2/icons/FullPermissionIcon/FullPermissionIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import type { IconProps } from '@/ui/v2/icons';
+import SvgIcon from '@mui/material/SvgIcon';
+
+function FullPermissionIcon({ sx, ...props }: IconProps) {
+  return (
+    <SvgIcon
+      width="20"
```

Comment:
I feel like this is too small for a 1920x1080 display (while this is correct, and I have left a comment in our designs I wonder if we should just bump it) on the other hand, it looks good in smaller resolutions

Reply:
The large version looks strange to me. Let's wait for @eugene-reuka 's response on this one.

Reply:
I also think that this is a relatively easy task. I wouldn't necessarily block the release by waiting for this to be resolved.